pkg/jobs/ansible/job_test.go

```diff
@@ -56,6 +56,7 @@ func getClusterCurator() *clustercuratorv1.ClusterCurator {
 					},
 				},
 			},
+			Inventory: ClusterName,
```

Comment:
As this is a generic function called to present the clusterCurator resource, is there a test where we skip inclusion and make sure its blank?

Reply:
added 2 test cases. one has inventory, another has no inventory.